explore/src/main/scala/explore/observationtree/ObsSummaryTable.scala

```diff
@@ -431,7 +431,8 @@ object ObsSummaryTable:
           innerContainerMod = ^.width := "100%",
           containerRef = resizer.ref,
           hoverableRows = rows.nonEmpty,
-          tableMod = ExploreStyles.ExploreTable |+| ExploreStyles.ObservationsSummaryTable,
+          tableMod =
+            ExploreStyles.ExploreTable |+| ExploreStyles.ObservationsSummaryTable |+| ExploreStyles.ExploreSelectableTable,
```

Comment:
Should this be rather `ExploreUnselectableTable`?

Reply:
I interpret it as "selectable rows".